app/controllers/manage_assessments/attachments_controller.rb

```diff
@@ -0,0 +1,10 @@
+module ManageAssessments
+  class AttachmentsController < ApplicationController
+    def show
+      @attachment = Attachment.find(params[:id])
+      authorize(@attachment)
+
+      redirect_to @attachment.file.expiring_url(Time.now + 60.seconds)
```

Comment:
I think this timeout is okay, but I see that we're repeating it a couple of times between the test and implementation. We're also having to jump through `.file` in both locations.

What do you think about adding `expiring_url` as a method directly on `Attachment`, that would call `file.expiring_url(Time.now + EXPIRE_TIMEFRAME)` (a constant we would set on the class). I think that would let us keep this logic contained and also keep us from having to do `attachment.file.`